app.js

```diff
@@ -1,14 +1,16 @@
 const express = require("express")
 const { getTopics, getEndpoints } = require("./controllers/topics_controllers")
-const { getArticleById, getArticles, getArticleCommentsById } = require("./controllers/articles_controllers")
+const { getArticleById, getArticles, getArticleCommentsById, postArticleCommentById } = require("./controllers/articles_controllers")
 const app = express()
 
+app.use(express.json())
 
 app.get("/api", getEndpoints)
 app.get("/api/topics", getTopics)
 app.get("/api/articles", getArticles)
 app.get("/api/articles/:article_id", getArticleById)
 app.get("/api/articles/:article_id/comments", getArticleCommentsById)
+app.post("/api/articles/:article_id/comments", postArticleCommentById)
 
 
```

Comment:
I think a new file for your error handling middleware would be nice at this point! It'll keep your app.js looking neat. 

Also, have a think about whether or not you need a switch statement at this point. Switch statements are usually used for complex conditional logic, and also you're responding with the same thing, so it might be pertinent to use an if statement instead. That being said, as it becomes more complicated (if it does!), then a switch statement would make perfect sense. (I'm pretty much just giving you my opinion, it makes no difference to the efficiency of the code!)